src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,42 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        if (toX < robot.getX()) {
+            while (robot.getDirection() != Direction.LEFT) {
+                if (robot.getDirection() == Direction.UP) {
+                    robot.turnLeft();
+                } else {
+                    robot.turnRight();
+                }
+            }
```

Comment:
let's make code more easier:
```suggestion
            while (robot.getDirection() != Direction.LEFT) {
                    robot.turnLeft();
            }
```